src/test/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/AgoraPermissionsSpec.scala

```diff
@@ -19,75 +19,79 @@ class AgoraPermissionsSpec extends ApiServiceSpec with BeforeAndAfterAll with Ag
-    assert(authorization.canRedact === true)
-    assert(authorization.canManage === true)
+    assert(authorization.canRead)
+    assert(authorization.canWrite)
+    assert(authorization.canCreate)
+    assert(authorization.canRedact)
+    assert(authorization.canManage)
   }
 
   "Agora" should "be able to construct authorizations using var arg permissions" in {
     val authorization = new AgoraPermissions(Read, Write, Redact)
-    assert(authorization.canRead === true)
-    assert(authorization.canWrite === true)
-    assert(authorization.canRedact === true)
-    assert(authorization.canCreate === false)
-    assert(authorization.canManage === false)
+    assert(authorization.canRead)
+    assert(authorization.canWrite)
+    assert(authorization.canRedact)
+    assert(!authorization.canCreate)
+    assert(!authorization.canManage)
   }
 
   "Agora" should "be able to remove permissions from authorization using var arg permissions" in {
     val authorization = new AgoraPermissions(All)
     val newAuthorization = authorization.removePermissions(Write, Create)
-    assert(newAuthorization.canRead === true)
-    assert(newAuthorization.canWrite === false)
-    assert(newAuthorization.canRedact === true)
-    assert(newAuthorization.canCreate === false)
-    assert(newAuthorization.canManage === true)
+    assert(newAuthorization.canRead)
+    assert(!newAuthorization.canWrite)
+    assert(newAuthorization.canRedact)
+    assert(!newAuthorization.canCreate)
+    assert(newAuthorization.canManage)
   }
 
   "Agora" should "be able to add permissions from an authorization using var arg permissions" in {
     val authorization = new AgoraPermissions(Read)
     val newAuthorization = authorization.addPermissions(Write, Create)
-    assert(newAuthorization.canRead === true)
-    assert(newAuthorization.canWrite === true)
-    assert(newAuthorization.canRedact === false)
-    assert(newAuthorization.canCreate === true)
-    assert(newAuthorization.canManage === false)
+    assert(newAuthorization.canRead)
+    assert(newAuthorization.canWrite)
+    assert(!newAuthorization.canRedact)
+    assert(newAuthorization.canCreate)
+    assert(!newAuthorization.canManage)
   }
 
   "Agora" should "be able to list admins" in {
     addAdminUser()
-    val adminUsers = AdminPermissionsClient.listAdminUsers
-    assert(adminUsers.length === 1)
-    assert(adminUsers.head === AgoraTestData.adminUser.get)
+    val adminUsers = runInDB { db => db.admPerms.listAdminUsers }
+    assert(adminUsers.length == 1)
+    assert(adminUsers.head == AgoraTestData.adminUser.get)
   }
 
   "Agora" should "be able to update the admin status of a user" in {
     val adminUser = AgoraTestData.adminUser.get
 
     // Set adminUsers's admin status false
-    AdminPermissionsClient.updateAdmin(adminUser, false)
-    var adminUsers = AdminPermissionsClient.listAdminUsers
-    assert(adminUsers.length === 0)
+    var noAdminUsers = runInDB { db =>
```

Comment:
I'm tempted to ask for separate transactions for update and list. However I don't see any problem with this as-is as long as we can assume that 1) the result of `listAdminUsers` will be the same before and after commit and 2) a commit failure will cause the test to fail. I believe that both of these assumptions are safe but I think it's worth being conscious of. I've seen cases where commit has side effects (checking constraints, invoking triggers, etc.) that, paired with poor transaction handling in tests, has led to false test success.